sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsBuffer.scala

```diff
@@ -94,6 +112,11 @@ trait RapidsBuffer extends AutoCloseable {
 
   val spillCallback: RapidsBuffer.SpillCallback
 
+  val unspillCallback: RapidsBuffer.UnspillCallback
+
+  /** The file offset of this buffer in bytes if it has already been written to disk. */
+  val fileOffset: Option[Long]
```

Comment:
The point of `RapidsBuffer` is to abstract details of different buffer types, and this is specific to a single buffer type (i.e.: one stored to disk).

Reply:
Removed.